lib/kafo/kafo_configure.rb

```diff
@@ -166,6 +166,12 @@ def logger
     def run(*args)
       started_at = Time.now
       logger.debug("Running installer with args #{args.inspect}")
+      if config.app[:verbose]
+        log_output_message = <<~MESSAGE.strip
+          Running installer with log based terminal output at level #{config.app[:verbose_log_level].upcase}. Use -l to change output verbosity to one of ERROR, WARN, NOTICE, INFO, or DEBUG.
+        MESSAGE
+        logger.notice(log_output_message)
```

Comment:
Sending this to the logger feels weird. It also means it ends up in the log files where it doesn't provide any value. Perhaps it should use regular `puts`?

Reply:
I don't think it adds any confusion if this message additionally shows up in the logs. The more common case is that the user will see it in their terminal output (under most common circumstances it will be the first thing they see). That gets me thinking,

1. Is it desired to see the time, stage, and log level for this message? I think that's mostly an aesthetic choice.

2. Should this message be displayed even if using `-l ERROR` for example? That would be an advantage of using `puts`, for the price of making the visual formatting of the output less consistent. But how common is that, and isn't the point of `-l ERROR` to not show output unless it's a problem?

Reply:
That's an interesting point. If the user passed in `-l` (or to be more precise, the value for `-l` is different from the default), do we need to inform the user about the `-l` option anyway? This goes both ways, so with `-l DEBUG` we can also leave it out.